rs/registry/canister/src/mutations/node_management/do_update_node_domain_directly.rs

```diff
@@ -153,7 +157,7 @@ mod tests {
                 .expect("failed to get the node operator id");
 
         // Assert setting domain name to Some() works
-        let new_domain = Some("invalid".to_string());
+        let new_domain = Some("_invalid".to_string());
```

Comment:
Why doesn't "invalid" trigger the failure now?  

If that's intended, then no change required.  I don't know enough about the domain the configuration affects to know.

Reply:
because "invalid" is actually a correct domain name. The reason why the test failed was because there was a check if there is a top level domain. 

However, the MR relaxes this check and it just validates that the domain name is correct without assuming any hierarchy